voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/SsmlDocumentBuilder.java

```diff
@@ -165,6 +177,26 @@ public SsmlDocumentBuilder(@NonNull List<String> ignoreTags) {
       getSentence().append(" strength=\"" + type + "\"");
     }
 
+    getSentence().append("/>");
+
+    return this;
+  }
+
+  /**
+   * Add a break element to the current sentence.
+   *
+   * @param time
+   *          The length of time to pause, nullable
+   * @return this, for method chaining
+   * @see #TIME_PATTERN
+   */
+  public @NonNull SsmlDocumentBuilder pause(@NonNull String time) {
+    if (ignoreTags.contains("break")) {
+      return this;
+    }
+
+    getSentence().append("<break");
+
     if (time != null && time.matches(TIME_PATTERN)) {
```

Comment:
This null check is redundant, because you have a `@NonNull` annotation on the parameter.